docs/extending.md

```diff
@@ -4,18 +4,22 @@
 
 # Extending
 
-Application extensibility is preformed via the root `app.extensions.json`,
+Application extensibility is preformed via the root `/src/assets/app.extensions.json`,
```

Comment:
performed

Reply:
Thanks for correction! I will fix that.

Reply:
@nbarithel you can see the live docs for this PR on Netlify: https://deploy-preview-543--aca-dev-docs.netlify.com/